
float64 rad
---
float64 ang_vel